src/clj/puppetlabs/puppetserver/certificate_authority.clj

```diff
@@ -102,6 +102,9 @@
   {:outcome (schema/enum :success :not-found :error)
    :message schema/Str})
 
+(def OIDMappings
+  {schema/Str schema/Keyword})
+
```

Comment:
No strong opinion here ... I'd be tempted to leave it as-is, but open to changing that as well.